### PR TITLE
Provide user ability to pass custom kubeconfig file path

### PR DESCRIPTION
**Pull request by smalleni**
Closes: #1

### CODE REVIEW

File: src/cerberus.py [hunk 1]
```diff
@@ -15,8 +15,12 @@
 requests.packages.urllib3.disable_warnings(InsecureRequestWarning)
```
Comment (chaitanyaenr) [inline src/cerberus.py]: We need to add a new line over here for the linters to pass ( expects two new lines before and after each function ). Run tox . from the base directory to make sure it passes. It's a manual process for now, @Yashashree1997 is working on adding a CI to do the checks for us.

Comment (jtaleric) [reply src/cerberus.py]: Do you plan to use Travis?

Comment (chaitanyaenr) [reply src/cerberus.py]: Yes, that's the plan: https://github.com/openshift-scale/cerberus/issues/8.

Comment (smalleni) [reply src/cerberus.py]: Weird. Running tox didn't give me any error.

Comment (chaitanyaenr) [reply src/cerberus.py]: I might have missed something, my bad. Thanks for checking.

**Comment by chaitanyaenr** [comment]
Thanks for the PR @smalleni.

**Pull request merged**